src/libstd/sys/common/condvar.rs

```diff
@@ -18,14 +20,14 @@ use sys::condvar as imp;
 /// condition variables. It is consequently entirely unsafe to use. It is
 /// recommended to use the safer types at the top level of this crate instead of
 /// this type.
-pub struct Condvar(imp::Condvar);
+pub struct Condvar(Box<imp::Condvar>);
```

Comment:
@alexcrichton There are two main reasons for moving the box. First, it seemed
preferable to me, to ensure that Condvar::new() returns fully initialized
object (especially now given that StaticCondvar is gone). Second, it didn't
require chasing all uses of Condvar::new() and inserting init nearby.

If you prefer to have boxes outside sys_common I can of course move them back.